frontend/packages/core/src/Input/select.tsx

```diff
@@ -279,8 +283,9 @@ export const Select = ({
       {label && <StyledInputLabel>{label}</StyledInputLabel>}
       {flatOptions.length && (
         <StyledSelect
+          multiple={multiple}
           id={`${name}-select`}
-          value={flatOptions[selectedIdx]?.value || flatOptions[selectedIdx].label}
+          value={selectedValues()}
           onChange={updateSelectedOption}
```

Comment:
If we are adding multiselect support should we change the onChange callback to pass in a list of strings instead of a single one? This way users don't need to manage state themselves.

Reply:
We are going to implement MultiSelect over Select to avoid version conflcits.